thingsboard_gateway/connectors/bacnet/application.py

```diff
@@ -31,10 +36,90 @@ def __init__(self, device_object_config: DeviceObjectConfig, indication_callback
     async def indication(self, apdu) -> None:
         self.__log.debug(f"(indication) Received APDU: {apdu}")
         await super().indication(apdu)
-        self.__indication_callback(apdu)
+        await self.__indication_callback(apdu)
 
     async def do_who_is(self, device_address):
         devices = await self.who_is(address=Address(device_address),
                                     timeout=self.__device_object_config.device_discovery_timeout)
         if len(devices):
             return devices[0]
+
+    async def get_object_identifiers(
```

Comment:
Could we move methods get_object_identifiers and get_device_objects to Device class? it looks more convenient to interact with it, don't you think so?